models/index.ts

```diff
@@ -1,70 +1,129 @@
+  return_likelihoods?: 'GENERATION' | 'ALL' | 'NONE';
 }
 
 export interface embed {
+  /** An array of strings for the model to embed. */
   texts: string[];
-  truncate?: "NONE" | "LEFT" | "RIGHT";
+  /** Specifies how the API will handle inputs longer than the maximum token length. */
+  truncate?: 'NONE' | 'LEFT' | 'RIGHT';
 }
 
 export interface chooseBest {
+  /** Used to query the options. */
   query: string;
+  /** Each string concatenates to the query. */
   options: string[];
-  mode: "PREPEND_OPTION" | "APPEND_OPTION";
+  /** One of PREPEND_OPTION|APPEND_OPTION to specify where the option string will be placed and
+   * how to compute the log-likelihood.
+   */
+  mode: 'PREPEND_OPTION' | 'APPEND_OPTION';
 }
 
-
-export type cohereParameters = | generate | embed | chooseBest;
+export type cohereParameters = generate | embed | chooseBest;
 
 /* -- responses -- */
 export interface text {
-  text: string,
-  likelihood: number,
+  /** Contains the generated text. */
+  text: string;
+  /** The sum of the log-likehoods of each token in the string. */
+  likelihood: number;
+  /** Only returned if `return_likelihoods` is not set to NONE. The likelihood. */
   token_likelihoods?: {
-    token: string,
-    likelihood?: number
-  },
-  [key: string]: any,
+    /** The token. */
+    token: string;
+    /** Refers to the log-likelihood of the token. The first token of a context will not
+     * have a likelihood.
+     * */
+    likelihood?: number;
+  };
+  [key: string]: any;
 }
 
 export interface embeddings {
-  embeddings: number[][],
-  [key: string]: any,
+  /** An array of embeddings, where each embedding is an array of floats. The length of the embeddings
+   * array will be the same as the length of the original texts array.
+   */
+  embeddings: number[][];
+  [key: string]: any;
 }
 
 export interface scores {
-  scores: number[],
-  tokens: string[][],
-  token_log_likelihoods: number[][],
-  [key: string]: any,
+  /** An array of floats corresponding to a score for each of the options, where a higher score
+   * represents a more likely query-option pair.
+   */
+  scores: number[];
+  /** An array of tokens corresponding to the tokens for each of the options. */
+  tokens: string[][];
+  /** An array of log likelihoods corresponding to the tokens of each of the options. */
+  token_log_likelihoods: number[][];
+  [key: string]: any;
 }
 
 export interface token_likelihoods {
-  likelihood: number,
+  /** The sum of the log-likehoods of each token in the string. */
+  likelihood: number;
   token_likelihoods: {
-    token: string,
-    likelihood?: number,
-  }
-  [key: string]: any,
+    /** The token. */
+    token: string;
+    /** Refers to the log-likelihood of the token. The first token of a context will not
+     * have a likelihood.
+     * */
+    likelihood?: number;
+  };
+  [key: string]: any;
 }
 
 export interface error {
-  message ?: string,
-  [key: string]: any,
+  /** Text explaining what went wrong */
```

Comment:
absolute tiniest of nits: missing a period 😭

Reply:
fixed loool